### PR TITLE
Fix deleted schedules

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

1. Remove the delete schedule worker job because the schedule should always exist if the function exists
2. Update the delete Function job to also delete schedules
3. Add a task to restore deleted schedules to fix functions in production.

## Test Plan

1. Deleted a function and checked the `console_schedules` table to confirm it's removed
2. Manually deleted rows from the `console_schedules` table and then ran the create-missing-schedules task to restore the deleted schedules

## Related PRs and Issues

None

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?

(The CHANGES.md file tracks all the changes that make it to the `main` branch. Add your change to this file in the following format)
- One line description of your PR [#pr_number](Link to your PR)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/Appwrite/Platform/Services/Tasks.php [hunk 1]
```diff
@@ -29,6 +30,7 @@ public function __construct()
             ->addAction(Doctor::getName(), new Doctor())
             ->addAction(Install::getName(), new Install())
             ->addAction(Maintenance::getName(), new Maintenance())
+            ->addAction(CreateMissingSchedules::getName(), new CreateMissingSchedules())
```
Comment (stnguyen90) [inline src/Appwrite/Platform/Services/Tasks.php:33]: I'm not sure what to call this task.

Comment (Meldiron) [reply src/Appwrite/Platform/Services/Tasks.php:33]: I like the name.

**Comment by Meldiron** [comment]
I did manual QA after my changes and a document was created successfully with the patch cli task.

**Comment by Meldiron** [comment]
https://user-images.githubusercontent.com/19310830/207534618-9f0330a4-ab43-44e2-b4e4-4b16b17bd61e.mp4

**Review by Meldiron — APPROVED** [review]

**Comment by Meldiron** [comment]
Closing in favour of https://github.com/appwrite/appwrite/pull/4860

We cannot delete schedule in functions as of right now. It makes sense but it will require more changes in schedule container before we can do this.

**Pull request closed without merging**